README.md

```diff
@@ -10,7 +10,7 @@
 
 ## Key Features
 
-* Backtesing exchange so you can try your trading strategies before using real funds.
+* Backtesting exchanges so you can try your trading strategies before using real funds.
```

Comment:
It should be *Backtesting exchange*, not exchanges.

Reply:
What does "Backtesting exchange" mean ...

Reply:
It means that, apart from Bitstamp and Binance, Basana also ships with a backtesting exchange class as documeted here: https://basana.readthedocs.io/en/latest/backtesting_exchange.html#basana.backtesting.exchange.Exchange